server/init/routes.js

```diff
@@ -83,6 +84,7 @@ export default (app) => {
       team: team.map(user => transformUser(user, req.user)),
       messages,
       sponsors,
+      eventConfig,
```

Comment:
huh, ok so there's just one api endpoint which returns everything at once? 
oh well, ok...

Reply:
Yes, this is not optimal, but I didn't want to change it in this PR